lib/resources/r4/observation.yaml

```diff
@@ -0,0 +1,116 @@
+---
+name: Observation
+field_name_base_url: http://hl7.org/fhir/R4/observation-definitions.html#Observation
+fields:
+
+- name: category
+  required: 'No'
+  cardinality: 0..*
+  type: CodeableConcept
+  description: A code that classifies the general type of observation being made.
+  binding:
+    description: Codes for high level observation categories.
+    terminology:
+    - display: Observation Category Codes
+      system: http://terminology.hl7.org/CodeSystem/observation-category
+      info_link: http://hl7.org/fhir/R4/valueset-observation-category.html
```

Comment:
info_links should be https

Reply:
https://github.com/cerner/fhir.cerner.com/pull/345/commits/85bb1a56bd4635bc292f0fbe1300685b9970981e#diff-53f5b7c6241dd914461705ddc9523c7eR3